package.json

```diff
@@ -9,9 +9,8 @@
     "build": "nx run-many --target=build",
     "test": "nx run-many --target=test",
     "e2e": "nx run-many --target=e2e",
-    "lint:spell": "cspell --no-progress '**/*.md'",
-    "lint:check": "npx eslint . ; npm run lint:spell",
-    "lint:fix": "npx eslint --fix .",
+    "lint:check": "node ./scripts/lint/lint-check.mjs",
+    "lint:fix": "node ./scripts/lint/lint-fix.mjs",
```

Comment:
@louis-bompart I use these scripts via the package.json

Reply:
My solution with the scripts work perfectly. If esling, cspell or prettier fail. The action fail as a whole

Reply:
twocents, might simpler to understand/follow to just have eslint . etc directly there.
a bit like: https://github.com/coveo/cli/blob/master/packages/cli/core/package.json#L177

Reply:
ah okay I guess && could work. I will try it